src/components/menuBar.tsx

```diff
@@ -41,16 +44,28 @@ export const MenuBar: React.FunctionComponent = () => {
     setSearch(e.target.value);
   };
 
+  const onSelectFilter = (name: string) => () => {
+    setSelectedButton(name);
+  };
+
+  const renderButton = (buttonName: string): JSX.Element => {
```

Comment:
why not make it a component rather than a function ? (it can still live in this file, out of the component definition)